src/formatDate.js

```diff
@@ -50,7 +58,62 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+  const dateStorage = {};
+  const result = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    switch (fromFormat[i][0]) {
+      case 'Y':
```

Comment:
чому перевіряєш тільки одну букву, давай будемо перевіряти всю строку
```suggestion
    switch (fromFormat[i]) {
      case 'YY':
```